fbfmaproom/fbfmaproom.py

```diff
@@ -1077,16 +1074,12 @@ def download_table():
 
     country_config = CONFIG["countries"][country_key]
     season_config = country_config["seasons"][season_id]
-    try:
-        month0 = abbrev_to_month0[issue_month]
-        issue_month_idx = season_config["issue_months"].index(month0)
-    except ValueError:
-        raise pingrid.NotFoundError("No forecasts issued in {issue_month}")
+    issue_month0 = abbrev_to_month0[issue_month]
```

Comment:
I know this was already merged, but I'm just curious why this exception raising was removed

Reply:
The line that could have raised the exception (line 1082) has been deleted, so there's no longer anything to catch here.

Empirically, it looks like we're still properly handling the case where the user selects a forecast that hasn't been issued yet (I see the alert). That's not the case that was being handled here. This was the case where somehow the UI requested the forecast for a month that the config file doesn't list as a valid month. Now that latter case will be handled by the same logic as the former.